### PR TITLE
CUDA and CUDnn support will need to be updated for Windows installation

### PR DESCRIPTION
**Pull request by zhwesky2010**
With the release of 1.5.2, CUDA and CUDnn support will also need to be updated for Windows installation.
Now，release of 1.5.2 support CUDA10,  and CUDA8 should match up cuDNN v7.1+.
MacHi 2019-09-06 23-39-06
MacHi 2019-09-06 23-39-30
MacHi 2019-09-06 23-34-33
MacHi 2019-09-06 23-35-40

### CODE REVIEW

File: doc/fluid/beginners_guide/install/compile/compile_Windows.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 * 如果您的计算机没有 NVIDIA® GPU，请编译CPU版的PaddlePaddle
 
 * 如果您的计算机有NVIDIA® GPU，并且满足以下条件，推荐编译GPU版的PaddlePaddle
-    * *CUDA 工具包8.0 配合cuDNN v7.1+, 9.0配合cuDNN v7.3+*
```
Comment (liupluswei) [inline doc/fluid/beginners_guide/install/compile/compile_Windows.md:15]: why delete cuDNN v7.1+ with CUDA 8.0?

File: doc/fluid/beginners_guide/install/install_Windows.md [hunk 2]
```diff
@@ -60,10 +60,10 @@
 * 如果您的计算机没有 NVIDIA® GPU，请安装CPU版的PaddlePaddle
 
 * 如果您的计算机有 NVIDIA® GPU，并且满足以下条件，推荐安装GPU版的PaddlePaddle
-    * *CUDA 工具包8.0配合cuDNN v7.1+， 9.0配合cuDNN v7.3+*
+    * *CUDA 工具包8.0/9.0/10.0配合cuDNN v7.3+*
     * *GPU运算能力超过1.0的硬件设备*
```
Comment (liupluswei) [inline doc/fluid/beginners_guide/install/install_Windows.md:64]: why delete cuDNN v7.1+ with CUDA 8.0?

File: doc/fluid/beginners_guide/install/install_Windows.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ Windows系统下有3种安装方式：
 * 如果是python2.7, 建议使用`python`命令; 如果是python3.x, 则建议使用`python3`命令
 
 
-* `python -m pip install paddlepaddle-gpu -i https://pypi.tuna.tsinghua.edu.cn/simple` 此命令将安装支持CUDA 8.0/9.0 cuDNN v7.3+的PaddlePaddle，如您对CUDA或cuDNN版本有不同要求，可用`python -m pip install paddlepaddle-gpu==[版本号] -i https://pypi.tuna.tsinghua.edu.cn/simple`或 `python3 -m pip install paddlepaddle-gpu==[版本号] -i https://pypi.tuna.tsinghua.edu.cn/simple`命令来安装，版本号请见[这里](https://pypi.org/project/paddlepaddle-gpu/#history), 关于paddlepaddle与CUDA, cuDNN版本的对应关系请见[安装包列表](./Tables.html/#whls)
+* `python -m pip install paddlepaddle-gpu -i https://pypi.tuna.tsinghua.edu.cn/simple` 此命令将安装支持CUDA 8.0/9.0/10.0 cuDNN v7.3+的PaddlePaddle，如您对CUDA或cuDNN版本有不同要求，可用`python -m pip install paddlepaddle-gpu==[版本号] -i https://pypi.tuna.tsinghua.edu.cn/simple`或 `python3 -m pip install paddlepaddle-gpu==[版本号] -i https://pypi.tuna.tsinghua.edu.cn/simple`命令来安装，版本号请见[这里](https://pypi.org/project/paddlepaddle-gpu/#history), 关于paddlepaddle与CUDA, cuDNN版本的对应关系请见[安装包列表](./Tables.html/#whls)
```
Comment (liupluswei) [inline doc/fluid/beginners_guide/install/install_Windows.md]: cuDNN v7.3+ maybe inaccurate here

File: doc/fluid/beginners_guide/install/install_Windows_en.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ There is a checking function below for [verifyig whether the installation is suc
 Notice:
 
 * The version of pip and the version of python should be corresponding: python2.7 corresponds to `pip`; python3.x corresponds to `pip3`.
-* `pip install paddlepaddle-gpu` This command will install PaddlePaddle that supports CUDA 8.0/9.0 cuDNN v7.3+, Currently, PaddlePaddle doesn't support any other version of CUDA or cuDNN on Windows.
+* `pip install paddlepaddle-gpu` This command will install PaddlePaddle that supports CUDA 8.0/9.0/10.0 cuDNN v7.3+, Currently, PaddlePaddle doesn't support any other version of CUDA or cuDNN on Windows.
```
Comment (liupluswei) [inline doc/fluid/beginners_guide/install/install_Windows_en.md]: same as the chinese version, confirm cuDNN version description with QA

**Comment by lanxianghit** [comment]
请在描述中明确一下具体要做哪些版本升级

File: doc/fluid/beginners_guide/install/install_Windows_en.md (hunk 4 above)
Comment (zhwesky2010) [reply doc/fluid/beginners_guide/install/install_Windows_en.md]: 已经修改

**Comment by zhwesky2010** [comment]
> 请在描述中明确一下具体要做哪些版本升级

已添加

**Review by lanxianghit — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**